asset-manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "files": {
+    "main.css": "./static/css/main.69eda509.chunk.css",
+    "main.js": "./static/js/main.3b76bc82.chunk.js",
+    "main.js.map": "./static/js/main.3b76bc82.chunk.js.map",
+    "runtime-main.js": "./static/js/runtime-main.77c289f4.js",
+    "runtime-main.js.map": "./static/js/runtime-main.77c289f4.js.map",
+    "static/js/2.48d6caae.chunk.js": "./static/js/2.48d6caae.chunk.js",
+    "static/js/2.48d6caae.chunk.js.map": "./static/js/2.48d6caae.chunk.js.map",
+    "index.html": "./index.html",
+    "precache-manifest.0db226bc3319860737f8d5bfa5901f9c.js": "./precache-manifest.0db226bc3319860737f8d5bfa5901f9c.js",
+    "service-worker.js": "./service-worker.js",
+    "static/css/main.69eda509.chunk.css.map": "./static/css/main.69eda509.chunk.css.map",
+    "static/js/2.48d6caae.chunk.js.LICENSE.txt": "./static/js/2.48d6caae.chunk.js.LICENSE.txt",
+    "static/media/1.png": "./static/media/1.89b0e751.png",
+    "static/media/2.png": "./static/media/2.c42482cc.png",
+    "static/media/3.png": "./static/media/3.184c7dbd.png",
+    "static/media/4.png": "./static/media/4.e11bbee5.png",
+    "static/media/Frame 1.png": "./static/media/Frame 1.7c53c6e8.png",
+    "static/media/about.png": "./static/media/about.4e44d31d.png",
+    "static/media/code.png": "./static/media/code.1587289c.png",
+    "static/media/contribute.png": "./static/media/contribute.552c3589.png",
+    "static/media/design_community.png": "./static/media/design_community.dc7f5974.png",
+    "static/media/documentation_training.png": "./static/media/documentation_training.9c56f1c3.png",
+    "static/media/getting-started.png": "./static/media/getting-started.ce74f5c4.png",
+    "static/media/instagram_logo.png": "./static/media/instagram_logo.3ce9c95b.png",
+    "static/media/landing.png": "./static/media/landing.1c27c683.png",
+    "static/media/linkedin-logo.png": "./static/media/linkedin-logo.5a653ec3.png",
+    "static/media/logo.png": "./static/media/logo.45cc4ce0.png",
+    "static/media/our-projects.png": "./static/media/our-projects.b19b61ad.png",
+    "static/media/our_events.png": "./static/media/our_events.1f92ca22.png",
+    "static/media/placeholder.png": "./static/media/placeholder.66c6a83d.png",
+    "static/media/project_management.png": "./static/media/project_management.f4a56233.png",
+    "static/media/research_outreach.png": "./static/media/research_outreach.e08739fb.png",
+    "static/media/testing.png": "./static/media/testing.1d609eba.png",
+    "static/media/twitter_logo.png": "./static/media/twitter_logo.b696dd8e.png"
+  },
+  "entrypoints": [
+    "static/js/runtime-main.77c289f4.js",
+    "static/js/2.48d6caae.chunk.js",
+    "static/css/main.69eda509.chunk.css",
+    "static/js/main.3b76bc82.chunk.js"
+  ]
+}
```

Comment:
Each file needs to have an empty line at the end of file as per the conventions.